src/components/message/message.twig

```diff
@@ -0,0 +1,21 @@
+{# 
+  @TODO
+  1. Generate an example of a message with a decorative icon.
+  #}
+
+{% set tagName = tagName | default('div') %}
+
+<{{tagName}} class="c-message{% if class %} {{class}}{% endif %}">
+  {% block content %}
+  <p>
+    {{label | default('Hi, I\'m a message!')}}
+  </p>
+  {% endblock %}
+  {% if includeClose %}
+  <button aria-label="close" class="c-message__close u-pad-none" type="button">
+    {% include '../../assets/icons/close.svg.twig' with {
+        class: 'c-icon'
+      } %}
+  </button>
+  {% endif %}
+</{{tagName}}>
```

Comment:
Missing newline at tend of file. (The editor config plugin manages this for me.)